.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
+Layout/SpaceInLambdaLiteral:
   Enabled: false
-Lint/StructNewOverride:
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: no_space
+Lint/BooleanSymbol:
+  Enabled: false
+Lint/StructNewOverride:
+  Enabled: false
 Lint/RaiseException:
+  Enabled: false
+Lint/SuppressedException:
+  Exclude:
+    - "spec/spec_helper.rb"
+Metrics/AbcSize:
+  Max: 25
+Metrics/CyclomaticComplexity:
   Enabled: true
+  Max: 12
+Metrics/MethodLength:
+  Enabled: false
+Metrics/ClassLength:
+  Enabled: false
 Metrics/BlockLength:
-  Exclude:
-    - "spec/**/*"
-    - "tmp/**/*"
+  Enabled: false
 Naming/FileName:
   Enabled: false
+# Naming/FileName:
+#   Exclude:
+#     - "lib/*-*.rb"
+Naming/MemoizedInstanceVariableName:
+  Enabled: false
+Naming/MethodName:
+  Enabled: false
+Naming/PredicateName:
+  Enabled: false
 Naming/RescuedExceptionsVariableName:
   PreferredName: exception
+Style/AccessModifierDeclarations:
+  Enabled: false
+Style/Alias:
+  Enabled: true
+  EnforcedStyle: prefer_alias_method
 Style/AndOr:
   Enabled: false
+Style/AsciiComments:
+  Enabled: false
+Style/BlockDelimiters:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Exclude:
+    - "spec/**/*_spec.rb"
+Style/ConditionalAssignment:
+  Enabled: false
+Style/DateTime:
+  Enabled: false
+Style/Documentation:
+  Enabled: false
```

Comment:
@timriley I don't agree on this. As an OSS project it's vital to document code.

Reply:
I agree but we could enforce it in a smarter way - ie via dedicated doc linting before a release. We use inch-ci to track doc coverage in dry-rb and rom-rb and I think it's better than having your CI builds fail because a fresh API you just started to experiment with doesn't have full doc coverage.

Reply:
@solnic We use Inch CI as well. But coverage != enforcing. 😉

Reply:
@jodosha my point was that we can enforce it at a later stage. Metric-related enforcements during early development is really slowing down the process. ie I don't want to spend X hours writing docs for a fresh API I'm adding in an unstable library that will most likely change a lot.

Reply:
@solnic Agreed, not asking for it 😄  My point is the following: adding just one line comment is enough. But I see your point, let's disable it. 😉